api/src/main/java/com/gentics/mesh/etc/config/ContentConfig.java

```diff
@@ -14,19 +14,36 @@
 public class ContentConfig implements Option {
 
 	public static final String MESH_CONTENT_AUTO_PURGE_ENV = "MESH_CONTENT_AUTO_PURGE";
+	public static final String MESH_GRAPH_BATCH_SIZE_ENV = "MESH_GRAPH_BATCH_SIZE";
```

Comment:
Name of the constant and name of the environment variable should be renamed to MESH_CONTENT_BATCH_SIZE, because this has nothing to do with "graph" (or does it?).